packages/core/bootstrap/package.json

```diff
@@ -12,7 +12,7 @@
     "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
     "prepack": "yarn build",
     "build": "tsc -b",
-    "dev:remotedev": "./remotedev.sh",
+    "dev:remotedev": "sh ./remotedev.sh",
```

Comment:
Don't need this, add a shebang to the shell script itself then it'll know what shell to execute